scripts/k8s-deployment-sim.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
+printf "\n\n\n*** [0] install server with tls template and HA in k8s\n"
+yq e '.global.agtK8Config.withPlugins.tls.enabled = true' -i k8s/example/values.yaml
+yq e '.global.agtK8Config.withPlugins.vln.enabled = false' -i k8s/example/values.yaml
```

Comment:
use k8s/agent+lber/value.yaml instead.

Reply:
Now lber tested with agent+lber/values.yaml and example/values.yaml used only for agent test.